packages/react-component-library/src/components/ProgressBar/partials/StyledProgressIndicator.ts

```diff
@@ -0,0 +1,14 @@
+import { color, type Spacing, spacing } from '@royalnavy/design-tokens'
+import styled from 'styled-components'
+import { StyledProgressBarProps } from './StyledProgressBar'
+
+interface StyledProgressIndicatorProps extends StyledProgressBarProps {
+  $height: Spacing
+}
+
+
+export const StyledProgressIndicator = styled.div<StyledProgressIndicatorProps>`
+  background-color: ${({ $isDisabled }) => $isDisabled ? color('neutral', '300') : color('action', '700')};
```

Comment:
I wonder if it should be color 500?

Reply:
![Screenshot 2025-04-16 at 08 32 08](https://github.com/user-attachments/assets/26b5cb7b-86bd-4478-9348-bdb4a45ea4d2)

Reply:
Split the difference - 600 - which is the same as the primary button